scripts/run_docker_build.sh

```diff
@@ -16,6 +16,7 @@ conda-build:
 
 always_yes: true
 show_channel_urls: true
+ssl_verify: false
```

Comment:
Will put it back, I have another file now that I run which is not part of the repository.